### ISSUE TITLE
[SC-IDE] auto-indent differs from style guidelines for multi-line method calls

### ISSUE THREAD

**Issue by mtmccrea** [supercollider/supercollider#4470]
## Environment

* SuperCollider version: 3.10.2
* Operating system: macOS 10.14

## Steps to reproduce

```supercollider
// run auto-indent on this
(
Button()
    .states_([["blorp", nil, nil]])
    .action_({
        "hey hey hey".postln
    });
)
```

## Expected vs. actual behavior
The [Code Style Guidelines](https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#method-calls) call for 
>The dot should be on the second line, not the first, and it should be indented one level. 

However, the auto-indent doesn't follow this rule and will undo any indentation you put in front of a method call on its own line.

#### Aside
While we're on the topic, thoughts on the semicolon being on it's own line after a multi-line chain of method calls?

```supercollider
(
Button()
    .states_([["blorp", nil, nil]])
    .action_({
        "hey hey hey".postln
    })
    ;
)
```
Would you consider this OK style or something to avoid? An advantage I find is that it makes it easy to keep adding more method call lines without missing the semicolon on a previous last line.

**Comment by brianlheim** [comment]
I would love for this to be configurable (and for the code style guidelines to be the default behavior)

> While we're on the topic, thoughts on the semicolon being on it's own line after a multi-line chain of method calls?

Looks horrible to me. :P Maybe easier for writing, but it's a pain to read and uses up precious vertical space.

**Comment by snappizz** [comment]
yeah the auto-indenter is a mess, and if someone made it nicer and more like standard text editors that would make a lot of users happy!

**Comment by mtmccrea** [comment]
>Looks horrible to me. :P 

Haha yea, I see that perspective. I've unconsciously formed the habit when I'm prototyping quickly, and I was updating some help docs and I wanted to check myself before letting this potentially bad style out into the world.
